easybuild/easyconfigs/g/gimkl/gimkl-1.6.0.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = "Toolchain"
+
+name = 'gimkl'
+version = '1.6.0'
```

Comment:
hmm, maybe a bigger bump is appropriate?

1.5.9 is *ancient*, and we're bumping the impi from 4.x to 5.x?

also, using `.0` leaves *no* room to downgrade a component later

how about something like

`2.2.5` (I know it looks weird and is kind of senseless, but it leaves room to play around with)

a maybe more sensible suggestion:

2.x (major bump because of new major impi)
2.11 (GCC .9 + impi .0 + imkl .2)
2.11.5 (room for downgrading if we need to)